### ISSUE TITLE
Crash of p2p daemon on windows

### ISSUE THREAD

**Issue by soffokl** [subutai-io/p2p#154]
1. Installed Subutai peer using windows installer-dev. 
2. Created environment 
3. Trying to connect to the container using tray application (EZ-SSH).
4. After clicking connect to the environment nothing happens for few seconds. 
5. Withing this short period of time executing `p2p debug` and getting panic on p2p daemon: 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x98 pc=0x4052c2]

goroutine 33 [running]:
panic(0x8b42a0, 0xc082008070)
        C:/Go/src/runtime/panic.go:464 +0x3f4
main.(*Procedures).Debug(0xc082009840, 0xc0822006a0, 0xc082200780, 0x0, 0x0)
        D:/Go/src/github.com/subutai-io/p2p/instance.go:335 +0x592
reflect.Value.call(0x878940, 0x908948, 0x13, 0x9478a0, 0x4, 0xc082221ed8, 0x3, 0x3, 0x0, 0x0, ...)
        C:/Go/src/reflect/value.go:435 +0x1214
reflect.Value.Call(0x878940, 0x908948, 0x13, 0xc082221ed8, 0x3, 0x3, 0x0, 0x0, 0x0)
        C:/Go/src/reflect/value.go:303 +0xb8
net/rpc.(*service).call(0xc082007d80, 0xc082007840, 0xc0821f49f0, 0xc082076300, 0xc0821ce6a0, 0x7d0640, 0xc0822006a0, 0x16, 0x7d0760, 0xc082200780, ...)
        C:/Go/src/net/rpc/server.go:383 +0x1c9
created by net/rpc.(*Server).ServeCodec
        C:/Go/src/net/rpc/server.go:477 +0x4a4
```

`p2p debug` gives following at that moment: 
```
C:\Users\user>p2p debug
[ERROR] Failed to run RPC request: read tcp 127.0.0.1:60436->127.0.0.1:52523: wsarecv: An existing connection was forcibly closed by the remote host.
```

**Comment by Dilshat** [comment]
Sometimes running p2p can crash on Linux too. No logs

**Comment by crioto** [comment]
@Dilshat @soffokl please check with latest version in dev

**Comment by crioto** [comment]
Fixed 4.0.0
